lib/inspec/objects/describe.rb

```diff
@@ -57,8 +57,8 @@ def initialize
       @variables = []
     end
 
-    def add_test(its, matcher, expectation)
-      test = Inspec::Describe::Test.new(its, matcher, expectation, false)
+    def add_test(its, matcher, expectation, negated = false)
```

Comment:
My only concern here is that we have a method with three positional args, adding a fourth optional positional arg.  I'd prefer adding an optional options hash, to allow for future growth in a flexible way:

```
def add_test(its, matcher, expectation, opts = {})
  test = Inspec::Describe::Test.new(its, matcher, expectation, opts[:negated])
  tests.push(test)
  test
end
```

Reply:
+1 on this approach; and @clintoncwolfe pointed out on https://github.com/inspec/inspec/pull/3577 that `only_if` is another property we want to manage, so we'd have at least two possible opts for a starter